CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Add: new simplifiedNormalized and simplifiedKeyvalues for attrsFormat (#4286)
```

Comment:
In doc/manuals/admin/database_model.md#csubs-collection the format description here:

> format: the format to use to send notification, possible values are JSON (meaning JSON notifications in NGSIv1 **legacy** format), **normalized**, **keyValues** and **values** (the last three used in NGSIv2 format).

should be expanded mentioning also the two new values.

Reply:
Added in commit https://github.com/telefonicaid/fiware-orion/pull/4298/commits/4bef2c1f3f62fb09d538271ff8e0554377b48d13